.github/workflows/release.yml

```diff
@@ -0,0 +1,12 @@
+on:
+  push:
+    tags:
+      - "v*"
+
+name: Release actions-generate-metadata Github Action
+jobs:
+  release:
+    name: Release actions-generate-metadata
+    runs-on: ubuntu-latest
+    steps:
+      - uses: technote-space/release-github-actions@v6
```

Comment:
I haven't seen this before, what's it do? (I read the readme on that action, just not clear why it's needed)

Reply:
can remove

Reply:
@kpenfound the action creates a release, please see here https://github.com/99/generate-metadata-action/releases  after tag is pushed  execution can be found here https://github.com/99/generate-metadata-action/actions/workflows/release.yml

Reply:
Removed it for now